src/Asm89/Twig/Lint/Command/LintCommand.php

```diff
@@ -91,6 +105,10 @@ protected function execute(InputInterface $input, CliOutputInterface $output)
 
         $errors = 0;
         foreach ($files as $file) {
+            if (true === $this->isFileInExcludeList($file, $excludeList)) {
+                $output->skip($template, $file);
```

Comment:
Up to you. It can be interesting for statistics. If it's not needed, I'll remove it.